br/backup-and-restore-tool.md

```diff
@@ -14,6 +14,12 @@ aliases: ['/docs/dev/br/backup-and-restore-tool/','/docs/dev/reference/tools/br/
 - Currently, TiDB does not support backing up and restoring partitioned tables.
 - Currently, you can perform restoration only on new clusters.
 - It is recommended that you execute multiple backup operations serially. Otherwise, different backup operations might interfere with each other.
+- BR only supports operations between clusters with the same `new_collations_enabled_on_first_bootstrap` [switch value](/character-set-and-collation.md#Collation support framework) because BR only backs up KV data. If the backup cluster and the recovery cluster use different sorting rules, the data verification fails. Therefore, when restoring the cluster, you need to make sure that the switch value query result of the `select VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME='new_collation_enabled';` statement is consistent with the query result of the backup. Then you can perform the restoration operation.
```

Comment:
```suggestion
- BR supports operations only between clusters with the same [`new_collations_enabled_on_first_bootstrap` value](/character-set-and-collation.md#collation-support-framework) because BR only backs up KV data. If the cluster to back up and the cluster to restore use different collations, the data validation fails. Therefore, before restoring a cluster, make sure that the switch value from the query result of the `select VARIABLE_VALUE from mysql.tidb where VARIABLE_NAME='new_collation_enabled';` statement is consistent with that during the backup process.
```